src/Microsoft.DotNet.Cli.Utils/IEnvironmentProvider.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public interface IEnvironmentProvider
```

Comment:
Originally I wanted IEnvironment but Environment conflicts with CoreFX.

I'm open to suggestions here.